rules_poetry/poetry.bzl

```diff
@@ -273,6 +357,14 @@ poetry = repository_rule(
             mandatory = False,
```

Comment:
Can we change this to be more descriptive like `python_interpreter_target` -> `python_interpreter_target_default`, also the docs on each should be updated

Reply:
updated [7aa2a09](https://github.com/mongodb-forks/rules_poetry/pull/1/commits/7aa2a09847cc9482b937f4381021a9cb45946821)